### ISSUE TITLE
fullscreen support?

### ISSUE THREAD

**Issue by bawn** [moaazsidat/react-native-qrcode-scanner#10]
How can I show camera view on fullscreen?

below code not work

```
  _topContent(){
    return(
      <View style={{height:0}}></View>
    )
  }
  _bottomContent(){
    return(
      <View style={{height:0}}></View>
    )
  }
  _onRead(){
    console.log(123213123123213);
  }
  render(){
    return(
      <View style={styles.container}>
        <QRCodeScanner
          style={{flex:1}}
          topContent={this._topContent()}
          bottomContent={this._bottomContent()}
          onRead={() => this._onRead()}
        />
      </View>
    )
  }
```

**Comment by moaazsidat** [comment]
@bawn sorry for the delay on this, was travelling, going to look into this this week.

**Comment by moaazsidat** [comment]
So a bit more context on this, currently the way the lib is set up, it renders a `<View>` and then passes the `topContent` prop as the child to render. There's two approaches to add this feature. One would be to have people be able to pass a container for the top and bottom like you demonstrate above. This would be a breaking change to the API. Another is to have styling props for the top, bottom and camera containers. and thus you'd be able to pass the relevant styles to make it full screen. 

See the full screen example here in the [PR](https://github.com/moaazsidat/react-native-qrcode-scanner/pull/14/): [https://github.com/moaazsidat/react-native-qrcode-scanner/pull/14/files#diff-e08eadef51fee10a880865e5370ee522]

**Comment by moaazsidat** [comment]
FIxed in https://github.com/moaazsidat/react-native-qrcode-scanner/pull/14

**Comment by bawn** [comment]
@moaazsidat Thanks, That's reasonable.

**Comment by moaazsidat** [comment]
Merged and shipped in 0.0.9 😄

**Comment by jneuendorf** [comment]
This works for me on (at least) iOS. Should this be contained in the README (maybe there should be a FAQ section)?
